tensorflow/compiler/xla/service/gpu/convolution_thunk.cc

```diff
@@ -57,9 +57,8 @@ Status ConvolutionThunk::ExecuteOnStream(const ExecuteParams& params) {
 
   auto op_profiler =
       params.profiler->MakeScopedInstructionProfiler(hlo_instruction());
-  TF_RETURN_IF_ERROR(RunCudnnConv(cudnn_call_,
-                                  absl::MakeSpan(operand_se_buffers),
-                                  result_buffer, scratch, params.stream));
+  TF_RETURN_IF_ERROR(RunGpuConv(cudnn_call_, absl::MakeSpan(operand_se_buffers),
```

Comment:
`conv_call_`?

Reply:
Actually I see, you would like to restrict the renaming, OK.

Reply:
Thanks for your understanding. From the past experience, those renaming PR are usually trivial to come up with, but take a long time to be applied because of the churning caused by rebase. Feel free to leave comments like this though, because they are very scoped and does not cause as much churning. I can collect those comments and file an independent PR in the end.